shared/message/behaviour/Look.proto

```diff
@@ -23,6 +23,9 @@ package message.behaviour;
 import "google/protobuf/duration.proto";
 import "Vector.proto";
 
+/**
+ * Details of where and when to look
+ */
 message Look {
```

Comment:
I feel like this file could do with more comments. I'm not really sure what angle and arcSize mean in this context. The angle I guess is the target angle for the head... But not sure how arcSize fits in.
From what I can tell, fixation, saccade and pan are different head movement types. Fixation being 'go to this position and stay', saccade 'go to this position but stop on the way for a set period of time the continue' but not sure how accurate that is, and pan being move towards the goal at a slow even pace? Only a guess tho.

Reply:
This file doesn't seem to be actually used in the code base. @Bidski is this file worth keeping?

Reply:
Without knowing the ideas behind how this was implemented, and without any references in the codebase of how it is used, I don't feel confident enough to add comments about this.

Reply:
Yeah, it's a tough one. I think behaviour needs a really good clean out. From what I understand behaviour is always left until RoboCup, and then it gets hacked together real quick, so no surprise it's a bit over the place. 
It can be something for the behaviour group to go through.

Reply:
@ElderNoSpace :p

Reply:
@KipHamiltons  Maybe after the director.

Reply:
I think this was stuff Jake was using for head behaviour. If it isnt being referenced anywhere in the codebase then remove it. If we need it again it will still be in the commit history

Reply:
Submitted #411